src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
```

Comment:
- [ ] [ HIGHLIGHT 🌟] Nice job implementing the OPTIONAL requirement 😸

Reply:
Thank you. 💪